src/terminfo/parser/compiled.rs

```diff
@@ -157,8 +158,22 @@ pub static stringnames: &'static[&'static str] = &[ "cbt", "_", "cr", "csr", "tb
     "OTG3", "OTG1", "OTG4", "OTGR", "OTGL", "OTGU", "OTGD", "OTGH", "OTGV", "OTGC", "meml", "memu",
     "box1"];
 
+fn read_harder<R: io::Read>(reader: &mut R, buffer: &mut [u8]) -> io::Result<usize> {
```

Comment:
Hm true. I'm not 100% sure that it's the best idea though, and I'd prefer to see how much mileage we can get without bothering looking at `Interrupted`.